_posts/2024-02-01-wildfly-glow.adoc

```diff
@@ -0,0 +1,504 @@
+WildFly server features that you are perhaps not aware of.
+
+WildFly Glow detailed http://docs.wildfly.org/wildfly-glow[documentation]. 
+
+=== WildFly Glow Features
+
+==== A set of Tooling
+
+* WildFly Glow CLI, a standalone tool to scan your deployment to produce a WildFly server, a WildFly Bootable JAR or a docker image.
+* Integration in WildFly Maven plugin 5.x, no more explicit feature-packs and layers in the plugin configuration.
+* WildFly Glow Arquillian Maven plugin to scan your tests to produce WildFly server required to execute your tests.
+
+==== Handling of High Availability
+
+* WildFly Glow allows you to enable the “ha” profile to produce an High Available WildFly server.
+
+==== Handling of 2 execution contexts
+
+* bare-metal (the default).
+* cloud, to execute on Openshift and/or Kubernetes. In addition to fine tune the server configuration for cloud execution, 
+it discovers and advertises the WildFly env variables usable to configure the server at startup.
+
+==== Handling of datasources
+
+* Identify missing datasources and suggest datasources that you can use to connect to databases.
+
+==== Knowledge of extra Feature-packs
+
+A centralized knowledge (located in the https://github.com/wildfly/wildfly-galleon-feature-packs[wildfly-galleon-feature-packs] github project) 
+of extra Galleon feature-packs compatible with WildFly. The WildFly features defined in these extra feature-packs 
+are included in the provisioned server when WildFly Glow detects a need for them. 
+
+The set of extra features supported are:
+
+* https://github.com/wildfly-extras/wildfly-cloud-galleon-pack[Cloud features]
+* Support for the major https://github.com/wildfly-extras/wildfly-datasources-galleon-pack[datasources] (PostgreSQL, MySQL, MariaDB, MSSQLServer, Oracle)
+* https://github.com/keycloak/keycloak[Keycloak] SAML
+* https://github.com/wildfly-extras/wildfly-grpc-feature-pack[GRPc]
+* https://github.com/wildfly-extras/wildfly-myfaces-feature-pack[MyFaces]
+* https://github.com/wildfly-extras/wildfly-graphql-feature-pack[Microprofile-GraphQL]
+
+==== Support for WildFly Preview
+
+WildFly Glow allows you to choose to provision a WildFly Preview server instead of a WildFly server.
+ 
+=== How WildFly Glow operates
+
+WildFly Glow is used from provisioning tooling: WildFly Glow command line tool or WildFly Maven Plugin (starting with version 5.0.0 Beta).
+
+Java classes and file descriptors located in your deployments are scanned to identify the required set of galleon layers. 
+
+* It leverages Galleon provisioning artifacts (Feature-packs and Layers).
+* It relies on rules included in each Galleon Layer.
+** Rules capture the content expected inside the deployment for the layer to be required.
+** Rules express the High Availability capability of a layer.
+** Rules classify some layers as `add-on` that can be explicitly included. add-ons are advertised according to the set of layers discovered in the deployments.
```

Comment:
add-ons should be plural. However, I always get a bit unsure when used with formatting whether this should be `add-ons` or `add-on`s :-)

Maybe 'A rule may classify a layer as an `add-on`...'?